.github/workflows/build.yml

```diff
@@ -6,25 +6,24 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v1
 
       - name: Use Go 1.13
         uses: actions/setup-go@v1
         with:
           go-version: 1.13.x
-          persist-credentials: false
 
-      - name: build game
+      - name: build
         run: |
           go get github.com/tmedwards/tweego
           export PATH=$PATH:$(go env GOPATH)/bin
-          tweego ia-cyoa.tw -o dist/index.html
+          tweego -f chapbook-1 ia-guide/ia-cyoa.tw -o dist/ia-cyoa.html
```

Comment:
curious if you know the specifics on `-f chapbook-1` and is `cyoa` create your own adventure?

Reply:
from what I was reading `chapbook-1` is like a system/format for tweego?

Reply:
yep, `-f` is the switch for `format`. Twine requires a format which allows you to do different things. The default is `sugarcube` but it's pretty basic, I chose `chapbook` because it allows you to have text input and better variable handling. FTR, tweego is just the compiler, so it take the Twine file (`.twee` or `.tw`), and uses the format (in this case, chapbook) to create the HTML.  As for my filenames, I've used `ia-cyoa` (choose your own adventure) is the interactive version, to differentiate it from `ia-static`, which is still to come.